docs/step-by-step/step-10.md

```diff
@@ -0,0 +1,213 @@
+# Step 10. Set up a proxy
+
+You're almost through! At this point, you should be pretty familiar with now Netdata works and how to configure it to
+your liking.
+
+In this step of the tutorial, we're going to add a proxy in front of Netdata. We're doing this for both improved
+performance and security, so we highly recommend following these steps. Doubly so if you installed Netdata on a
+publicly-accessible remote server.
+
+> ❗ If you installed Netdata on the machine you're currently using (aka on `localhost`), and have been accessing Netdata
+> at `http://localhost:19999`, you can skip this step of the tutorial, as you can't set up a proxy for a service that's
+> running locally.
```

Comment:
This is a bit disingenuous, as it absolutely is possible to set up a proxy on the local system, it just makes essentially no sense unless you're also exposing Netdata externally.

I would suggest changing it from 'you can't set up a proxy for a service that's running locally' to 'there is no benefit to setting up a proxy for a service that's running locally' (though that's not entirely true either, as you can make Netdata use a UNIX socket and then use a proxy to password protect the dashboard even for local users).